src/main/java/com/google/devtools/build/lib/bazel/rules/cpp/BazelCppSemantics.java

```diff
@@ -78,15 +78,15 @@ public void finalizeCompileActionBuilder(
       RuleErrorConsumer ruleErrorConsumer) {
     CcToolchainProvider toolchain = actionBuilder.getToolchain();
     if (language == Language.CPP) {
+      CppConfiguration cppConfig = configuration.getFragment(CppConfiguration.class);
```

Comment:
Let's also add a warning at the top of this method:

`ruleErrorConsumer.ruleWarning("Include scanning is turned on. This feature is unsupported.");`

Reply:
Indeed. Let's have the warning once per invocation by adding it in IncludeScanningModule.beforeCommand when the experimental flag is passed.

Reply:
added

Reply:
Thanks Benjamin. Once the other threads where Janak is helping are resolved I can merge.